package.json

```diff
@@ -46,6 +46,7 @@
     "commander": "2.9.0",
     "event-stream": "3.3.4",
     "express": "4.15.2",
+    "express-handlebars": "^3.0.0",
```

Comment:
Webpack on the server, I'm hoping @xPaw did not mean that 😅.

Apart from that, it's usually a good practice in webapps to pre-render as much as you can so `index.html` comes back from server with all it needs already there, so moving rendering on the client would not necessarily be a bonus.